flyway-core/src/main/java/org/flywaydb/core/internal/dbsupport/DbSupportFactory.java

```diff
@@ -131,7 +131,7 @@ public static DbSupport createDbSupport(Connection connection, boolean printInfo
         }
 
 		//Sybase ASE support
-        if (databaseProductName.startsWith("ASE")) {
+        if (databaseProductName.startsWith("Adaptive Server Enterprise")) {
```

Comment:
From what I tested in ASE 15.2, it returns ASE. I think the name is different in 15.7?